catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreCleanupHandler.java

```diff
@@ -103,6 +104,13 @@ public void handleEvent(Event event) {
                         .map(Metacard::getId)
                         .collect(Collectors.toList());
                 if (!idsToDelete.isEmpty()) {
+                    LOGGER.debug(
```

Comment:
Wrap this in LOGGER.isDebugEnabled to avoid iterating over all the metacards if we'll ignore this statement anyways.